### PR TITLE
standardized naming for metas

### PR DESCRIPTION
**Pull request by hotdoy**
fields for metas (pages-metas & site-config) are standardized.
title, description, image.
Global var for site-config are now...
site-title
site-description
site-image

### CODE REVIEW

File: workspace/utilities/site-variables.xsl [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:exslt="http://exslt.org/common"
+	exclude-result-prefixes="exslt">
 
-	<!-- Default site name with current language -->
-	<xsl:variable name="site-name">
-		<xsl:call-template name="default-site-name" />
-	</xsl:variable>
+<xsl:variable name="data" select="/data" />
```
Comment (nitriques) [inline workspace/utilities/site-variables.xsl]: I do not see the point of having $data.

File: workspace/utilities/site-variables.xsl [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+<xsl:variable name="params" select="$data/params" />
```
Comment (nitriques) [inline workspace/utilities/site-variables.xsl]: Same here.

File: workspace/utilities/site-variables.xsl (hunk 1 above)
Comment (hotdoy) [reply workspace/utilities/site-variables.xsl]: We use a lot of exslt node sets and not having to care about having access to /data is way easier for everyone.

File: workspace/utilities/site-variables.xsl (hunk 2 above)
Comment (fhamon) [reply workspace/utilities/site-variables.xsl]: I think it's a "cleaner" way to access the data. That way all our pointers inside the various templates starts with `$` and not a mix of `$` & `/`.

Comment (dominic-blain) [reply workspace/utilities/site-variables.xsl]: Yes, but mainly for the reason @hotdoy mentions above. When we create a RSL, we lose access to the parents and its can break things or just make things harder (think Date template, who needs /data )

File: workspace/utilities/lib/master-title.xsl [hunk 3]
```diff
@@ -38,11 +38,11 @@
 		<xsl:param name="lg" select="$url-language" />
 
 		<xsl:choose>
-			<xsl:when test="count($config/site-titre/item) != 0">
-				<xsl:value-of select="$config/site-titre/item[@lang=$lg]"/>
+			<xsl:when test="count($config/titre/item) != 0">
+				<xsl:value-of select="$config/titre/item[@lang=$lg]"/>
```
[the remark this answers is not in the document]
Comment (nitriques) [reply workspace/utilities/lib/master-title.xsl]: why ?

File: workspace/utilities/site-variables.xsl [hunk 4]
```diff
@@ -45,6 +54,9 @@
 </xsl:variable>
 <xsl:variable name="twitter-card-domain" select="$config/twitter-site-domain" />
 
+<!-- MONEY DECIMAL FORMAT -->
```
[the remark this answers is not in the document]
Comment (nitriques) [reply workspace/utilities/site-variables.xsl]: Indeed, can you please remove ?

File: workspace/utilities/lib/master-title.xsl (hunk 3 above)
[the remark this answers is not in the document]
Comment (PascalPiche) [reply workspace/utilities/lib/master-title.xsl]: The reverse language was used originally for the meta attribute. This comit remove it. 
<link ~~title="{$page-title}"~~

File: workspace/utilities/site-variables.xsl (hunk 4 above)
[the remark this answers is not in the document]
Comment (hotdoy) [reply workspace/utilities/site-variables.xsl]: yep

File: workspace/utilities/site-variables.xsl [hunk 5]
```diff
@@ -1,13 +1,22 @@
...
+	exclude-result-prefixes="exslt">
 
-	<!-- Default site name with current language -->
-	<xsl:variable name="site-name">
-		<xsl:call-template name="default-site-name" />
-	</xsl:variable>
+<xsl:variable name="data" select="/data" />
+<xsl:variable name="params" select="$data/params" />
+<xsl:variable name="config" select="$data/site-config/entry[1]" />
+<xsl:variable name="pages" select="$data/pages/page" />
```
[the remark this answers is not in the document]
Comment (PascalPiche) [reply workspace/utilities/site-variables.xsl]: Still a problem here...

[the remark this answers is not in the document]
Comment (PascalPiche) [reply workspace/utilities/site-variables.xsl]: ah no.. sorry fixed. thanks

File: workspace/utilities/lib/master-title.xsl [hunk 6]
```diff
@@ -38,11 +38,11 @@
 		<xsl:param name="lg" select="$url-language" />
 
 		<xsl:choose>
-			<xsl:when test="count($config/site-titre/item) != 0">
-				<xsl:value-of select="$config/site-titre/item[@lang=$lg]"/>
+			<xsl:when test="count($config/titre) != 0">
```
Comment (PascalPiche) [inline workspace/utilities/lib/master-title.xsl]: Not needed anymore. the string length will be enought

**Pull request merged**